### ISSUE TITLE
Django2 problem

### ISSUE THREAD

**Issue by Bersam** [modlinltd/django-advanced-filters#84]
Hello there, I've installed latest django-advanced-filters using following command:
pip install -e "git://github.com/modlinltd/django-advanced-filters.git@develop#egg=django-advanced-filters"
but still i face following error:
```
    from django.core.urlresolvers import reverse
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```
any idea what could be the problem?

**Comment by greenmark** [comment]
You need replace all occurrences of:
`from django.core.urlresolvers import resolve`
to 
`from django.urls import resolve`

**Comment by asfaltboy** [comment]
I believe this is fixed in PR #63, I'm not sure how you installed latest version from develop and still see it, try to make sure you remove existing package and install without using cache.
Otherwise, if you can wait, I will hopefully release a new version with the latest changes in develop by the end of the month.
